doc/Alerting/Templates.md

```diff
@@ -281,7 +280,6 @@ Alert-ID: {{ $alert->id }} <br>
 Rule: @if ($alert->name) {{ $alert->name }} @else {{ $alert->rule }} @endif <br>
 @if ($alert->faults) Faults:
 @foreach ($alert->faults as $key => $value)
-#{{ $key }}: {{ $value['string'] }}<br>
```

Comment:
Why did you remove this one?

Reply:
The lines starting with hash tags cause the parser to throw a syntax error.  If they are supposed to be comments and you would like to keep the line then I can amend it to use the {{-- comment --}} which I think is the laravel comment formatting.